apps/src/common.rs

```diff
@@ -89,6 +89,8 @@ pub struct Client {
 
     pub http_conn: Option<Box<dyn HttpConn>>,
 
+    pub id: u64,
```

Comment:
Is there a more descriptive term than just `id` that we could use?

Reply:
Maybe `client_id`? It is used as an intermediate identifier between Connection IDs and actual client's connections.